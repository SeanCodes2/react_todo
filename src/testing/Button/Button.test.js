import React from 'react'
import Button from './Button'
import renderer from 'react-test-renderer'
import { cleanup, render, screen } from '@testing-library/react'

afterEach(cleanup)

it('Button Renders correctly', () => {
    const tree = renderer.create(<Button />).toJSON()

    expect(tree).toMatchSnapshot()
})

