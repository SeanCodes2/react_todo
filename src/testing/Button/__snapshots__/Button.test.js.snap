// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Renders correctly 1`] = `
<div
  className="text-center"
>
  <button
    className="btn btn-warning"
    data-testid="button"
  />
</div>
`;
